fabric-data-generation-api-v1/src/testmod/java/net/fabricmc/fabric/test/datagen/DataGeneratorTestEntrypoint.java

```diff
@@ -238,7 +236,7 @@ public void generateTranslations(TranslationBuilder translationBuilder) {
 			translationBuilder.add(SIMPLE_BLOCK, "Simple Block");
 			translationBuilder.add(new Identifier(MOD_ID, "identifier_test"), "Identifier Test");
 			translationBuilder.add(EntityType.ALLAY, "Allay");
-			translationBuilder.add(EntityAttributes.GENERIC_ARMOR, "Generic Armor");
+			translationBuilder.add(EntityAttributes.GENERIC_ARMOR.value(), "Generic Armor");
```

Comment:
I think we need to add this, like we did with `CreativeModeTab`
```java
default void add(RegistryEntry<EntityAttribute> key, String translation) {
    add(key.value());
}
```
And then no breaking change
```java
translationBuilder.add(EntityAttributes.GENERIC_ARMOR, "Generic Armor");
```

Reply:
Ah, yes good spot. Thanks

Reply:
Maybe also remove `add(EntityAttribute, String` and leave only `add(RegistryEntry<EntityAttribute> key, String translation)`

Reply:
Yep, thats what ive done 👍